example/helloWorld/src/helloworld.cpp

```diff
@@ -0,0 +1,177 @@
+/**
+ * \file
+ * Copyright 2014-2015 Benjamin Worpitz
```

Comment:
better keep it in the top line, we are all lazy people and will forget to update multiple positions ;)